File: bigdata-spark_2.11/day08-spark/src/main/scala/cn/itcast/spark/kafka/source/Test01.scala
package cn.itcast.spark.kafka.source

import scala.util.Random

/**
 * @author by FLX
 * @date 2021/6/23 0023 16:42.
 */
object Test01 {
  def main(args: Array[String]): Unit = {
    val random = new Random()
    val callIn: String = "1890000%04d".format(random.nextInt(10000))

    println(callIn)
  }
}

File: course-flink/flink-day03/src/test/java/cn/itcast/test/Test01.java
package cn.itcast.test;

import java.util.Random;

/**
 * @author by FLX
 * @date 2021/7/19 0019 20:54.
 */
public class Test01 {
    public static void main(String[] args) {
        Random random = new Random();
        int i = random.nextInt(2) + 1;

        System.out.println(i);
    }
}
